docs/SPEC.md

```diff
@@ -182,21 +182,22 @@ When several durations are specified together, larger units must appear before s
 
 Durations represent a length of time.
 Lengths of time are dependent on specific instants in time they occur and as such, durations do not represent a fixed amount of time.
-No amount of seconds is equal to a day, as days vary in their number of seconds.
-No amount of days is equal to a month, as months vary in their number of days.
-A duration consists of three basic time units: seconds, days and months.
+Durations are a vector of the necessary components to increase or decrease a time value.
```

Comment:
I removed the language describing the implementation. Durations are a vector and that's all we really need to specify here.

Reply:
I think it's useful to describe rationale for the design here, or link to somewhere that does.  The concept of math on durations and times feels so intuitive, but there are a lot of complex details.  Readers may want to know why such limitations exist.

Reply:
I'll re-add the bit about months varying in days. I'm hesitant to include additional examples because I want to free up the implementation. While the implementation is likely going to be tracking only months and nanoseconds, I don't want the spec to specify that.

Reply:
I updated the language here. I think it's much better now as I think it describes the rationale.